svm/src/transaction_processor.rs

```diff
@@ -228,20 +239,49 @@ impl<FG: ForkGraph> TransactionBatchProcessor<FG> {
             sanitized_txs,
             check_results,
             error_counters,
-            &self.fee_structure,
             account_overrides,
             &program_cache_for_tx_batch.borrow(),
         );
```

Comment:
github won't let me comment above.

Could you explain the interface here? I understand that's what was there, but it doesn't make sense to me in the context where we have conflicting (or really even shared tbh) accounts.

If we have two txs both loading account A, we'd be loading it twice instead of once. Feels like we should be creating the `unique_loaded_accounts` as part of this to me. wdyt?

Reply:
I agree, what you suggesting is that instead of loading the accounts multiple times then iterating over them and constructing a `unique_loaded_accounts`, we should make `unique_loaded_accounts` part of the `load_transaction_accounts` to avoid multiple account loads.